refocus-bdk-server.js

```diff
@@ -278,23 +278,24 @@ module.exports = (config) => {
    *
    * @param {Express} app - App stream so we can push events to the server
    * @param {String} token - Socket Token needed to connect to Refocus socket
+   * @param {String} botId - Id of the bot
    */
-  function refocusConnectSocket(app, token) {
+  function refocusConnectSocket(app, token, botId) {
```

Comment:
@jgraff2 we shouldn't need this is a parameter, this function should have access to a variable called botName in it's scope

Reply:
Isn't this passed in on line 1146?